hz.ps1

```
@@ -1541,7 +1541,7 @@ function hz-build {
         $options.constants = $options.constants.Replace(";", "%3B") # escape ';'
     }
 
-    $branchName = git symbolic-ref --short HEAD
+    $branchName = git name-rev --name-only HEAD
```

It doesn't produce same result. Instead, could you add `-q` to suppress the error?

---

> Instead, could you add `-q` to suppress the error?
No, because it still sets the error level / exit code to `1`.

---

Well, this won't work. `name-rev` doesn't give the full refs path which includes the _release_ directory.

---

> Well, this won't work. `name-rev` doesn't give the full refs path which includes the _release_ directory.

Ok, good point.

b72e3a9de98017e484299a293e8dc35318b37413